R/queue.R

```diff
@@ -12,12 +13,14 @@ Queue <- R6::R6Class(
     next_health_check = NULL,
 
     initialize = function(queue_id = NULL, workers = 2,
-                          cleanup_on_exit = workers > 0,
+                          stop_workers_on_exit = workers > 0,
```

Comment:
i think this is still a bit dangerous because the default is creating 2 workers and it doesnt discriminate between different workers so if theres some on prod and we accidentally spawn new ones, itll still bring down all the workers on exit, `worker_manager$id` has all the ids of the workers spawned by this queue so i think we should also only kill those workers

Reply:
Thanks, that's a good shout. I've made those changes too now. Will not start any workers by default, and it will only cleanup the workers the `queue` creates itself.